setup.py

```diff
@@ -374,9 +374,9 @@ def make_relative_rpath(path):
         ],
     },
     data_files=[
-        'scripts/fixup_binary.py',
+        ('scripts', ['scripts/fixup_binary.py']),
```

Comment:
i wonder what the change here does

Reply:
I'm fixing a warning from using a blank install directory: https://docs.python.org/3/distutils/setupscript.html#installing-additional-files